pkg/proxy/ipvs/ipset.go

```diff
@@ -30,39 +30,63 @@ const (
 
 	// KubeLoopBackIPSet is used to store endpoints dst ip:port, source ip for solving hairpin purpose.
 	KubeLoopBackIPSet = "KUBE-LOOP-BACK"
```

Comment:
Looks much better now. One more small suggestion. Now that the comment for the variable is the same as the value for the comment variable, maybe we can structure it like so:

```
KubeLoopbackIPSetComment = "Kubernetes endpoints dst ip:port, source ip for solving hairpin purpose"
KubeLoopBackIPSet = "KUBE-LOOP-BACK"
```

Basically use the value of the comment variable as the comment for the actual IPSet. In other words, no need for "//" style comments.

Reply:
@rramkumar1 Thanks for reviewing! I forgot to mention that I made those variables unexported since the exported variables need comments (golint checks it).